<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.walletapplication.payme.model.exceptions.GlobalWalletExceptionTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2024-01-15T15:52:36" hostname="nishi" time="0.004">
  <properties/>
  <testcase name="fullConstructor()" classname="com.walletapplication.payme.model.exceptions.GlobalWalletExceptionTest" time="0.001"/>
  <testcase name="constructorWithMessage()" classname="com.walletapplication.payme.model.exceptions.GlobalWalletExceptionTest" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
